packages/core/admin/admin/src/containers/App/index.js

```diff
@@ -34,7 +34,13 @@ import PrivateRoute from '../PrivateRoute';
 import routes from './utils/routes';
 import { makeUniqueRoutes, createRoute } from '../SettingsPage/utils';
 
-const queryClient = new QueryClient();
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      refetchOnWindowFocus: false,
```

Comment:
Why do we need this refetch now?

Reply:
To prevent react query to refetch when the window is focused: when going to another tab and coming back